db/migrate/20200403194501_rapsearch_to_rapsearch2.rb

```diff
@@ -0,0 +1,5 @@
+class RapsearchToRapsearch2 < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :pipeline_runs, :completed_rapsearch_chunks, :completed_rapsearch2_chunks
```

Comment:
wow this is a pretty thorough rename!

in general, I'd be cautious about renaming any db col. you need to understand the downstream dependents really well, which could even include analytics reports! in this case, I think its safe

Reply:
`completed_rapsearch_chunks` has no usages in the code base. I assume it was to do with the autoscaling but I am hesitant to drop it for historical reasons. I do want to ensure we rename it everywhere though.